### DISCUSSION TITLE
[ASR]事前学習済みモデル(CSJ)について

### DISCUSSION THREAD

**Discussion by t-ozawa123** [espnet/espnet#4380] [Q&A]
こんにちは。ASRの社内研究をしており参考にさせていただいております。
以下URLで公開されております、CSJの事前学習済みモデルに関する質問です。
https://zenodo.org/record/4065140#.YodF--jP1GM

事前学習の際、コーパスにはフィラー（F:あー）などが含まれていると思います。
このようなテキストデータについて何らかの加工を施しているのでしょうか？
もしくはモデルの出力後に別途フィラーの判別が必要になるという事でしょうか？

その他、学習データ内に記号等、発音がないテキストが含まれていると
再現の仕方も含め、扱いに困ってしまいます。
一般論でも構いませんのでご教示いただけますと幸いです。

**Comment by sw005320** [comment]
> このようなテキストデータについて何らかの加工を施しているのでしょうか？

コーパスによりますが基本は何もしていません
喋ったそのままを認識するのが音声認識のタスクなので

> もしくはモデルの出力後に別途フィラーの判別が必要になるという事でしょうか？

モデルの出力後に形態素解析を行なって、POSタグなどからフィラーを除けばいいと思います
他にもフィラーの識別器を作るとか色々手はあると思います
